test/core/gprpp/inlined_vector_test.cc

```diff
@@ -486,6 +486,18 @@ TEST(InlinedVectorTest, PopBackAllocated) {
   EXPECT_EQ(sz - 1, v.size());
 }
 
+TEST(InlinedVectorTest, Resize) {
+  const int kInlinedSize = 2;
+  InlinedVector<UniquePtr<int>, kInlinedSize> v;
+  // Size up.
+  v.resize(5);
+  EXPECT_EQ(5UL, v.size());
+  EXPECT_EQ(nullptr, v[4]);
+  // Size down.
+  v.resize(1);
```

Comment:
Before sizing down, I suggest populating one of the elements that is going to be destroyed, to prove that the elements' dtors are properly called when this happens.

Reply:
Sorry, what does this mean?

Reply:
I suggest adding something like the following right before this line:

```
v[4] = MakeUnique<int>(5);
```

Then, when we resize down to 1, the code should free that element.  If it doesn't, there will be an asan failure due to a memory leak.

Reply:
I still don't understand the benefit of this change. 

Why would populating an additional element help prove the dtor is properly called during sizing down? Even without this additional element, the code should free the elements with index from 1 to 4. If it doesn't, there would already be an asan failure.

Reply:
Currently, the elements in indexes 1 through 4 are all null, so calling the dtor does nothing.  If the dtor was not being called, this test would not fail.  Changing one of those elements to be non-null will cause the test to fail if the elements' dtor is not called.

Reply:
I see.. I was thinking about the dtor of `UniquePtr` itself.

Reply:
It *is* the dtor of `UniquePtr<>` that we're talking about calling, but that dtor won't actually do anything that would be caught by asan if the `UniquePtr<>`'s value is null.

Reply:
Done.

Reply:
Yeah, that is what I meant to say.